api/controllers/site-branch-config.js

```diff
@@ -145,6 +145,7 @@ module.exports = wrapHandlers({
     const { id, site_id: siteId } = params;
 
     const site = await Site.forUser(user)
+      // eslint-disable-next-line testing-library/await-async-queries
```

Comment:
**non-blocking** I think this rule only applies to test files anyway so we should remove it from the code lint globally (rather than line by line). Also I'm not sure why it's only showing up in a few places since `find*` is a common pattern

ref: https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/await-async-queries.md#rule-details

Reply:
I scoped it down to just `test/`, `*.spec.js|x`, `*.test.js|x` and removed the disable